src/lib/types/user.ts

```diff
@@ -18,9 +18,9 @@ export interface UserData {
 
 export interface IUser {
     id: number;
-    name?: string;
-    username?: string;
-    email?: string;
+    name: string;
+    username: string;
+    email: string;
```

Comment:
Just adapting to the User class definition

Reply:
Do we really always have all of name, username, and email? I thought (at least) one of these was often unset? Does it default to an empty string?

Reply:
It's a fair question and to the best of my knowledge this type is plain wrong... it should be something like: `either have username or email`. I tried doing that but I couldn't. We can try, but ultimately that's what the class definition is (which was already wrong), so I thought this could be a problem for later as we're not changing the interface.